resources/sdk/purecloudkotlin/gradleDeploy.sh

```diff
@@ -0,0 +1,16 @@
+BUILD_DIR=$1
+IS_NEW_RELEASE=$2
+
+echo "BUILD_DIR=$BUILD_DIR"
+echo "IS_NEW_RELEASE=$IS_NEW_RELEASE"
+
+if [ ! "$IS_NEW_RELEASE" = "true" ]
+then
+	echo "Skipping maven deploy"
+	exit 0
+fi
+
+# CD to build dir
+cd $BUILD_DIR
+
+./gradlew artifactoryPublish
```

Comment:
Per our discussion, move this to the internal config.

Reply:
Totally missed that. That file was accidentally put there, I'll delete it